src/bootstrap/bootstrap.py

```diff
@@ -390,5 +398,9 @@ def main():
     env["BOOTSTRAP_PARENT_ID"] = str(os.getpid())
     rb.run(args, env)
 
+    end_time = time()
+
+    print("Build completed in %s" % format_build_time(end_time - start_time))
```

Comment:
Yeah, concatenation is not ideal, IMHO the best option is simply `print("...", timedelta)` but that requires adding the `__future__` import and that's probably out of scope. Perhaps I'll prepare a PR for that, since it seems #33036 was abandoned.